bin/vspec

```diff
@@ -28,7 +28,7 @@ if [[ $# != 0 && "${args[$#-1]%%*.t}" == '' ]]
 then
   "${args[$#-1]}"
 else
-  "${VSPEC_VIM:-vim}" -u NONE -i NONE -N -n -e -s --cmd "source $0-bootstrap.vim" "$@" 2>&1 |
+  "${VSPEC_VIM:-vim}" -u NONE -i NONE -N -n -e -s --cmd "source $0-bootstrap.vim" -- "$@" 2>&1 |
```

Comment:
Please update also `t/environment-variable.t` to pass all tests.

Reply:
Done.